src/Analyzer.Core.BuiltInRuleTests/README.md

```diff
@@ -3,46 +3,35 @@ This project is designed for quickly adding tests to verify the correctness of b
 
 ## Test Setup
 A test (or tests) for a given rule consists of 2 parts (each part is described in more detail below):
-1. A JSON configuration describing the test(s).
-2. One or more test ARM templates to analyze.  These templates are in a directory specifically for testing.  The results of the analysis are compared with the test configuration to assert correctness of the rule.
+1. A new directory, containing a new JSON configuration describing the test(s) - the name of both the directory and the JSON configuration must match the ID of the rule it will test.
+2. One or more test ARM templates to analyze.  These templates must be within the same directory the test configuration file resides in.  The results of analyzing these templates are compared with the test configuration to assert correctness of the rule.
 
 ### JSON Configuration
-To create a set of tests for a rule, a new JSON file is created in the *Tests* directory.  **The name of the file must be the same as the `name` property of the JSON rule**, with ".json" as the file extension.
+To create a set of tests for a rule, a new directory and JSON file are created.  **The name of the directory and file must be the same as the `id` property of the JSON rule**, with ".json" as the file extension.
 
-For example, a JSON rule like the following:
-``` js
-{
-    "name": "SuperSecurityCheck",
-    "description": "...",
-    "recommendation": "...",
-    "helpUri": "...",
-    "evaluation": {
-      ...
-    }
-}
-```
-... would have tests defined in a configuration file named **"Tests/SuperSecurityCheck.json"**.
+For example, to write tests for rule ID *TA-000001*, a test configuration file must be located at *TA-000001/TA-000001.json* within the *Tests* directory.
 
 The JSON test configuration has the following schema:
 ``` js
 [
     {
-        "Template": "Name of template file analyzed (without file extension).  Template must be in the 'TestTemplates' directory.",
+        "Template": "Name of template file analyzed.  Template must be within the same directory as the test configuration file.",
```

Comment:
Also, you should add that this now needs a file ending. Before it did not.

Reply:
It didn't "need" the file extension before because this was how *.badtemplate* was enforced.  Now that extension isn't enforced, so it makes more sense to include the extension here.  I guess previously it was obvious from the example, but I removed that in this change.  I'll go ahead and add that back.

Reply:
Actually I misread the example I removed, it wasn't for the configuration.  I was explicit about not specifying the file extension previously because that's not typically what's expected.  Now this is in line with what's usually expected, so I simply removed the text.